docs/user/installation.rst

```diff
@@ -10,17 +10,22 @@ Express Install
 
 The fastest way to set up dependencies for garage is via running the setup script.
 
+Clone this repo and navigate to its directory.
```

Comment:
Which repo?

These docs appear at rlgarage.readthedocs.io. The user doesn't necessarily have any context to the GitHub repo.